content/en/real_user_monitoring/operations_monitoring.md

```diff
@@ -0,0 +1,248 @@
+---
+title: Operations Monitoring
+description: Monitor critical technical operations within user-facing features to identify exactly when and why users fail to complete key workflows.
+private: true
+further_reading:
+- link: '/monitors/create/types/real_user_monitoring/'
+  tag: 'Documentation'
+  text: 'Learn about RUM'
+---
+
+{{< callout url="" btn_hidden="true" header="Join the Preview!">}}
+Operations Monitoring is in Preview.
+{{< /callout >}}
+
+## Overview
+
+{{< img src="/real_user_monitoring/operations_monitoring/operations-monitoring-1-temp.png" alt="Operations tab under RUM > Performance Monitoring" style="width:80%;" >}}
+
+In Datadog RUM, a feature represents a major user-facing area of your application like checkout, login, or search. Each feature includes operations, which are the critical technical steps that make the experience work. 
+
+- Business teams use **features** to track and improve user conversion.
+- Engineering teams use **operations** to monitor and minimize technical failures that impact key user moments. 
+
+For example, the checkout experience of an eCommerce platform is a feature. Within it, operations might include entering payment details, saving a payment method, and completing a purchase. After the SDK has been instrumented, Datadog RUM measures each operation's performance, including execution volume, completion rate, and failure rate. Measuring operations' health enables you to identify exactly when and why users may not convert in your feature.
+
+The following table shows additional example features and their associated feature operations by industry.
+
+| Industry       | Feature  | Feature Operations                                                                                                               |
+|----------------|----------|----------------------------------------------------------------------------------------------------------------------------------|
+| Social network | Profile  | Users can load their profile <br> Users can upload a picture <br> Users can update their status                                  |
+| E-Commerce     | Checkout | Users can enter payment details <br> Users can save their payment method <br> Users can pay                                      |
+| Streaming      | Search   | Users can find results for their search <br> Users can load the description of a title <br> Users can start watching the trailer |
+| CRM            | Quote    | Users can start a new quote <br> Users can add line items to the quote <br> Users can send a quote to recipients                 |
+
+## Prerequisites
```

Comment:
The separate headings in this section made it a little hard for me to understand the relationship between them. For example, if you're using the browser SDK but you don't have RUM without Limits, you can't use operations...I think? I wasn't immediately sure if it was an either/or situation, or if you need RUM without Limits to do anything. If that's the case, maybe it should go before the API options?